src/store/actions/commentsActions.js

```diff
@@ -0,0 +1,82 @@
+import { toast } from 'react-toastify';
+import axiosConfig from '../../axiosConfig';
+import {
+  FETCH_AN_ARTICLE,
+  FETCH_ARTICLE_COMMENTS,
+  FETCH_AN_ARTICLE_404,
+  CREATE_COMMENT_SUCCESS,
+  CREATE_REPLY_SUCCESS,
+} from './actionTypes';
+
+const getItem = (actionType, data) => ({
+  type: actionType,
+  payload: data,
+});
+
+export const fetchAnArticle = slug => (dispatch) => {
+  return axiosConfig
+    .request({
+      method: 'get',
+      url: `articles/${slug}/`,
+    })
+    .then((response) => {
+      dispatch({
+        type: FETCH_AN_ARTICLE,
+        payload: response.data.Article,
+      });
+    })
+    .catch((error) => {
+      (getItem(FETCH_AN_ARTICLE_404, error));
+    });
+};
+
+export const fetchArticleComments = slug => (dispatch) => {
+  return axiosConfig
+    .request({
+      method: 'get',
+      url: `articles/${slug}/comments/`,
+    })
+    .then((response) => {
+      dispatch(getItem(FETCH_ARTICLE_COMMENTS, response.data.Comments));
+    })
+    .catch((error) => {
```

Comment:
Please try and dispatch an error/failure action to the store because that state might be important for use when trying to implement certain conditions ... The toast is not so reliable for us developers because it disappears after a few seconds but the state is more consistent. I would suggest you create a `FETCH_ARTICLE_COMMENTS_FAILURE` action or something similar